docs/quick_start.rst

```diff
@@ -84,135 +113,247 @@ on multiple inputs.
-desired instances and it is structured as below:
+in the root directory. It represents the simexpal configuration read in to run
+the experiments on the desired instances. The file is structured as below:
 
 .. literalinclude:: ../examples/sorting/experiments.yml
    :linenos:
    :language: yaml
-   :caption: Example of experiments.yml file
+   :caption: Example of an experiments.yml simexpal configuration file.
 
-The structure of this file will be better explained later in the guide.
-At this point, our ``sorting`` directory looks like this:
-::
+The structure of the configuration file will get more attention later on. At
+this point, our ``sorting`` directory looks like this: 
 
-   sorting
-   ├── sort.py
-   ├── experiments.yml
-   └── instances
-       └── random_500.list
+::
 
-After having completed this steps, we can start using simexpal to run our experiments.
-A complete list of experiments and their status can be seen by:
+  sorting
+  ├── sort.py
+  ├── experiments.yml
+  └── instances
+      └── random_500.list
+
+To add the instance to be used in our experiments, we must add a local file with
+it's name *random-500* (no extension) to our instances stanza:
+
+.. code-block:: yaml  
+
+  instances:
+    - generator:
+        args: ['./generate.py', '--seed=1', '1000']
+      items:
+        - uniform-n1000-s1
+    - generator:
+        args: ['./generate.py', '--seed=2', '1000']
+      items:
+        - uniform-n1000-s2
+    - generator:
+        args: ['./generate.py', '--seed=3', '1000']
+      items:
+        - uniform-n1000-s3
+    - repo: local
+      items:
+        - random-500
+
+  experiments:
+    - name: insertion-sort
+      args: ['./sort.py', '--algo=insertion-sort', '@INSTANCE@']
+      stdout: out
+    - name: bubble-sort
+      args: ['./sort.py', '--algo=bubble-sort', '@INSTANCE@']
+      stdout: out
+
+After having completed this step, we can start using simexpal to run our
+experiments including the new instance. The @-Variable ``@INSTANCE@`` will be
+use for all instances. A complete list of experiments and their status can be
+seen by:
 
 .. code-block:: bash
 
-   $ simex experiments list
+   $ simex e list
 
 The color of each line represents the status of the experiment:
 
-- Green: finished
-- Yellow: running
-- Red: failed
-- Default: not executed
+.. role:: green
+.. role:: yellow
+.. role:: red
+
+.. raw:: html
+
+    <style> .red {color:red} </style>
+    <style> .yellow {color:yellow} </style>
+    <style> .green {color:green} </style>
+
+- green :green:`🮆` represents *finished*
+- yellow :yellow:`🮆` represents *running*
+- red :red:`🮆` represents *failed*
+- and the default color represents *not executed*
 
 Experiments can be launched with:
 
 .. code-block:: bash
 
-   $ simex experiments launch
+   $ simex e launch --launch-through=fork
 
 This instruction will launch the non executed experiments on the local machine.
```

Comment:
Could you add a hyphen here, too? "non-executed"

Reply:
I did :heavy_check_mark: